src/containers/main/InvestContainer/InvestMarket.tsx

```diff
@@ -92,6 +97,26 @@ export const InvestMarket: FC<InvestMarketProps> = ({ handleStart, handleStop })
 		).toFixed(toFixed);
 	}
 
+	const handleSetAggregatedRewards = (reward_amount: number) => {
+		setAggregatedRewards((aggregatedRewards) => [...aggregatedRewards, reward_amount]);
+	};
+
+	//This needs to be viewed by someone, issue is I need to clean up aggregatedRewardsArr after I do the math, but updating it retriggers useEffect and app gets stuck in loop
+
+	useEffect(() => {
```

Comment:
This useEffect has a param of aggregatedRewards.

But in order to make sure it works properly after doing a state change I need to set aggregatedRewards to [0] which retriggers useEffect